x-pack/legacy/plugins/code/public/style/_layout.scss

```diff
@@ -195,3 +195,9 @@
 .code-symbol-container {
   white-space: nowrap;
 }
+
+.codeBreadcrumbs {
+  .euiPopover__anchor {
```

Comment:
Use the `anchorClassName` prop rather than overwriting EUI.

Reply:
I was just looking at this. `EuiBreadcrumbs` won't actually pass that className through to the popover, although it probably should

Reply:
Since this is for a margin, my guess a wrapper element for the button itself can work?